doc/eredis_cluster.md

```diff
@@ -122,7 +122,7 @@ redis_result() = <a href="#type-redis_simple_result">redis_simple_result()</a> |
 
 
 <pre><code>
-redis_simple_command() = [<a href="#type-anystring">anystring()</a>]
+redis_simple_command() = [<a href="#type-anystring">anystring()</a> | integer()]
```

Comment:
when is the command and integer?

Reply:
In scan/4, the spec says some args are integers. The full command is a list. The first element is never an integer, but args may be, for convenience and are sent as strings. The encoder actually accepts more types but integer is the only arg type which makes sense.